File: Genral_All_Controls/PanAndZoomExample/PanAndZoomExample/PanImageControl.cs
using System.ComponentModel;
using Telerik.WinControls;

namespace PanAndZoomExample
{
    [ToolboxItem(true)]
    public class PanImageControl : RadControl
    {
        private PanImageElement panImageElement;

        protected override void CreateChildItems(RadElement parent)
        {
            base.CreateChildItems(parent);

            this.panImageElement = new PanImageElement();
            parent.Children.Add(this.panImageElement);            
        }

        public PanImageElement PanImageElement
        {
            get { return panImageElement; }
        }
    }
}

File: Genral_All_Controls/PanAndZoomExample/PanAndZoomExampleVB/PanImageControl.vb
Imports Telerik.WinControls
Imports System.ComponentModel

<ToolboxItem(True)>
Public Class PanImageControl
    Inherits RadControl

    Private _panImageElement As PanImageElement

    Protected Overrides Sub CreateChildItems(ByVal parent As RadElement)
        MyBase.CreateChildItems(parent)
        Me._panImageElement = New PanImageElement()
        parent.Children.Add(Me._panImageElement)
    End Sub

    Public ReadOnly Property PanImageElement As PanImageElement
        Get
            Return _panImageElement
        End Get
    End Property
End Class